src/main/kotlin/LottoGameController.kt

```diff
@@ -0,0 +1,105 @@
+        result.onSuccess { input ->
+            return input
+        }.onFailure { error ->
+            println(error.message)
+            return null
+        }.also {
+            throw IllegalStateException(UNKNOWN_ERROR)
+        }
+    }
+
+    private fun getPaymentMoney(): PaymentMoney {
+        val input = inputView.inputPaymentMoney()
+        val validationResult = PaymentMoneyValidation().checkPaymentMoney(input)
+        return handleInputResult(validationResult) ?: getPaymentMoney()
+    }
+
+    private fun getLottoCount(totalLottoCount: Count): LottoCount {
+        val input = inputView.inputManualLottoCount()
+        val validationResult = ManualLottoCountValidation().checkManualLottoCountValidation(input, totalLottoCount)
+        return handleInputResult(validationResult) ?: getLottoCount(totalLottoCount)
+    }
+
+    private fun getManualLotteries(manualLottoCount: Count): List<Lotto> {
+        return List(manualLottoCount.value) {
+            val input = inputView.inputManualLotto()
+            val validationResult = LottoValidation().checkLottoValidation(input)
+            handleInputResult(validationResult) ?: return getManualLotteries(manualLottoCount)
+        }
+    }
+
+    private fun getWinningLotto(): Lotto {
+        val input = inputView.inputWinningLotto()
+        val validationResult = LottoValidation().checkLottoValidation(input)
+        return handleInputResult(validationResult) ?: getWinningLotto()
+    }
+
+    private fun getWinningNumbers(winningLotto: Lotto): WinningNumber {
+        val input = inputView.inputBonusNumber()
+        val validationResult = BonusNumberValidation().checkBonusNumberValidation(input, winningLotto)
+        return handleInputResult(validationResult) ?: getWinningNumbers(winningLotto)
+    }
+
+    private fun printAutoLotteries(lotteries: List<Lotto>) {
+        outputView.printAutomaticLotteries(lotteries)
+    }
+
+    private fun getRankStatistics(
+        manualLotteries: List<Lotto>,
+        autoLotteries: List<Lotto>,
+        winningNumber: WinningNumber
+    ): RankStatistics {
+        val rankStatistics = RankStatistics()
+        manualLotteries.forEach { lotto ->
+            val rank = lotto.getRank(winningNumber)
+            rankStatistics.updateRankCount(rank)
+        }
+        autoLotteries.forEach { lotto ->
+            val rank = lotto.getRank(winningNumber)
+            rankStatistics.updateRankCount(rank)
+        }
+        return rankStatistics
+    }
+
+    private fun showResult(rankStatistics: RankStatistics) {
+        outputView.printRankStatistics(rankStatistics)
+        getProfitRate(rankStatistics)
+    }
+
+    private fun getProfitRate(rankStatistics: RankStatistics) {
+        val profitRate = rankStatistics.getProfitRate()
+        val isLoss = rankStatistics.isProfitable(profitRate)
+        outputView.printProfitRate(profitRate, isLoss)
+    }
+
+    companion object {
+        const val UNKNOWN_ERROR = "알수 없는 오류입니다"
```

Comment:
```suggestion
        private const val UNKNOWN_ERROR = "알수 없는 오류입니다"
```
클래스 내부에서만 사용하는 값이라면 private 접근자를 붙여보면 어떨까요? (이하 관련 내용 동일)